ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/generator/WorkBasketResultFieldsGenerator.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.ccd.sdk.generator;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+import uk.gov.hmcts.ccd.sdk.ConfigBuilderImpl;
+import uk.gov.hmcts.ccd.sdk.JsonUtils;
+import uk.gov.hmcts.ccd.sdk.types.Role;
+import uk.gov.hmcts.ccd.sdk.types.WorkBasketResult;
+import uk.gov.hmcts.ccd.sdk.types.WorkBasketResult.WorkBasketResultBuilder;
+import uk.gov.hmcts.ccd.sdk.types.WorkBasketResultField;
+
+public class WorkBasketResultFieldsGenerator {
+
+  public static <T, R extends Role, S> void generate(File root, String caseType,
```

Comment:
Not sure you need the type parameters here?

Reply:
you're right, copy and paste 😅